lib/src/ast/event.dart

```diff
@@ -153,20 +156,19 @@ class ParsedEventAst extends TemplateAst
   @override
   int get quotedValueOffset => valueToken?.leftQuote?.offset;
 
-  /// Offset of `(` prefix in `(eventName.postfix)`.
+  /// Offset of `(` prefix in `(eventName.reductions)`.
   @override
   int get prefixOffset => prefixToken.offset;
 
-  /// Offset of `)` suffix in `(eventName.postfix)`.
+  /// Offset of `)` suffix in `(eventName.reductions)`.
   @override
   int get suffixOffset => suffixToken.offset;
 
-  /// Name `postfix` in `(eventName.postfix)`; may be `null` to have no value.
+  /// Name `filters` in `(eventName.ctrl.shift.a)`; may be empty.
   @override
-  String get postfix {
+  List<String> get reductions {
     final split = _nameWithoutParentheses.split('.');
-    assert(split.length < 2);
-    return split.length > 1 ? split[1] : null;
+    return split.getRange(1, split.length).toList();
```

Comment:
split.subList(1) is probably a more succinct way.

Reply:
Ooh, very nice. Done!